presto-main/src/main/java/com/facebook/presto/operator/aggregation/DoubleRegressionAggregation.java

```diff
@@ -84,7 +84,8 @@ public static void regrIntercept(@AggregationState RegressionState state, BlockB
     public static void regrSxy(@AggregationState RegressionState state, BlockBuilder out)
     {
         double result = getRegressionSxy(state);
-        if (Double.isFinite(result)) {
+        double count = getRegressionCount(state);
+        if (Double.isFinite(result) && Double.isFinite(count) && count > 0) {
```

Comment:
If the count is not finite, should it throw?

Reply:
If the count is not finite, the result should be null, which is consistent with the behavior of the regr_count function.